azure-functions/acbs-function/helpers/date.is-epoch.test.js

```diff
@@ -72,16 +67,6 @@ describe('isEpoch', () => {
       mockValue: '8640000000000000', // https://262.ecma-international.org/5.1/#sec-15.9.1.1
       expected: true,
     },
-    {
```

Comment:
These tests were passed by the moment implementation but I think making them work is a waste of time